include/eve/concept/basic.hpp

```diff
@@ -0,0 +1,50 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/kumi.hpp>
+#include <type_traits>
+#include <cstdint>
+#include <cstddef>
+
+namespace eve
+{
+  template<typename T>
+  concept builtin_vectorizable  =     std::is_arithmetic_v<T>
+                                  && !std::is_same_v<T,bool>
+                                  &&  sizeof(T) <= 8;
```

Comment:
I don't like the name tbh. 

Given that is_integral is already a list, do we maybe want to just list everything?

https://github.com/llvm/llvm-project/blob/50de7f1e7787bf327d1d04c4daab97c1c180b58c/libcxx/include/__type_traits/is_integral.h#L22

Reply:
I fear we may miss some std::intx_t on some architecture if we use a list

Reply:
We should miss it. We don't support char for example. We have a very specific list.

Reply:
OK